lib/shopify_app/controller_concerns/login_protection.rb

```diff
@@ -45,8 +49,10 @@ def current_shopify_session
           is_online: user_session_expected?,
         )
       rescue ShopifyAPI::Errors::CookieNotFoundError
+        ShopifyApp::Utils::Logger.warn("CookiesNotFound for current shopify session")
         nil
       rescue ShopifyAPI::Errors::InvalidJwtTokenError
+        ShopifyApp::Utils::Logger.warn("Invalid Jwt token for current shopify session")
```

Comment:
Should rescued errors be warns?

Reply:
Since these are 'handled' errors I would treat them as `info` or `debug`.